Testing/functional/tests/cypress/integration/ui/timeline/filter.js

```diff
@@ -133,6 +150,56 @@ describe("Filters", () => {
             "[data-testid=timeline-clinical-document-disclaimer-alert]"
         ).should("not.exist");
     });
+
+    it("Verify diagnostic imaging record alert appears when only imaging reports is selected", () => {
+        cy.get(
+            "[data-testid=linear-timeline-diagnostic-imaging-disclaimer-alert]"
+        ).should("not.exist");
+
+        cy.get("[data-testid=filterDropdown]").click();
+        cy.get("[data-testid=DiagnosticImaging-filter] input").click({
+            force: true,
+        });
+        cy.get("[data-testid=btnFilterApply]").click();
+        cy.get("[data-testid=btnFilterApply]").should("not.exist");
+
+        cy.get(
+            "[data-testid=linear-timeline-diagnostic-imaging-disclaimer]"
+        ).should("be.visible");
+
+        cy.get("[data-testid=filterDropdown]").click();
+        cy.get("[data-testid=HealthVisit-filter] input").click({ force: true });
+        cy.get("[data-testid=btnFilterApply]").click();
+
+        cy.get(
+            "[data-testid=linear-timeline-diagnostic-imaging-disclaimer]"
+        ).should("not.exist");
+    });
+
+    it("Verify cancer screening record alert appears when only imaging reports is selected", () => {
```

Comment:
```suggestion
    it("Verify BC Cancer screening alert appears only when BC Cancer screening is selected", () => {
```

Reply:
Similar to the Diagnostic imaging alert comment, they're not equivalent.